hack/helm_vars/wire-server/values.yaml.gotmpl

```diff
@@ -342,6 +342,10 @@ galley:
           defaults:
             status: disabled
             lockStatus: locked
+        cells:
```

Comment:
According to the [documentation](https://wearezeta.atlassian.net/wiki/spaces/PAD/pages/1800962144/2025-03-19+Wire+Cells+implementation+in+wire-server), this should be locked and disabled by default.